daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1007,7 +1007,7 @@ object SBuiltin {
       val stakeholders = observers union signatories
       val contextActors = machine.ptx.context match {
         case ContextExercises(ctx) => ctx.actingParties union ctx.signatories
-        case ContextRoot => machine.committers.toList.toSet
+        case ContextRoot => machine.committers
```

Comment:
this Set is immutable, not sure why `toList.toSet` was there

Reply:
:+1: 

The explanation is very simple. If you looked the history of committers, you could saw that  `commiters` use to be a `Option[Party]`.